addons/web/static/src/js/views/list/list_controller.js

```diff
@@ -400,6 +400,26 @@ var ListController = BasicController.extend({
             return result;
         }, []);
         return new Promise((resolve, reject) => {
+            const saveRecords = () => {
+                this.model.saveRecords(this.handle, recordId, validRecordIds, fieldName)
+                    .then(async () => {
+                        this._updateButtons('readonly');
+                        const state = this.model.get(this.handle);
+                        const params = { keepWidths: true };
+                        // We need to check the current multiEdition state here in case
+                        // the selection is changed
+                        const multiEdit = this.renderer.isInMultipleRecordEdition(recordId);
```

Comment:
can't you rely on validRecordIds.length?

Reply:
I don't think so, the multiEdit boolean should reflect the multi-editable state (more than one record selected) of the list whether or not the changes are valid.